spec/integration/food_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe 'Food index page', type: :feature do
+  let(:user) { User.create(name: 'test', email: 'test@gmail.com', password: 'password') }
+
+  before :each do
+    sign_in user
+  end
+
+  it 'displays the link to create new food' do
+    visit foods_path
+    expect(page).to have_content('New Food')
+  end
+
+  it 'should create and show a food' do
+    # create a food
+    visit foods_path
+    click_link 'New Food'
+    fill_in 'Name of food', with: 'test food'
+    fill_in 'Measurement unit', with: 'test unit'
+    fill_in 'Price', with: '1.0'
+    fill_in 'Quantity', with: '1'
+    click_button 'Create Food'
+
+    # check and see if the food is created properly
+    expect(page).to have_content('test food')
+    expect(page).to have_content('test unit')
+    expect(page).to have_content('1.0')
+    expect(page).to have_content('1')
+  end
+
+  it 'should edit and show a food' do
+    # create a food
+    visit foods_path
+    click_link 'New Food'
+    fill_in 'Name of food', with: 'test food'
+    fill_in 'Measurement unit', with: 'test unit'
+    fill_in 'Price', with: '1.0'
+    fill_in 'Quantity', with: '1'
+    click_button 'Create Food'
```

Comment:
- [ ] These two tests are failing, Please make sure to fix this and all tests should pass without any errors.

![image](https://github.com/sylvester-wamaya/Capstone-Project-Recipe/assets/83803414/a5aa7364-dba0-4b99-ab55-675466e11412)